symphony-bdk-spring/symphony-bdk-core-spring-boot-starter/src/main/java/com/symphony/bdk/spring/config/BdkCoreConfig.java

```diff
@@ -21,11 +22,14 @@
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 
+import java.util.Optional;
+
 /**
  * Configuration and injection of the main BDK/Core classes as beans within the Spring application context.
  */
 @Slf4j
 public class BdkCoreConfig {
+  private static final String BEARER_AUTH = "bearerAuth";
```

Comment:
This constant could be a static property of the `OAuthentication` class, that is used in both core and spring-boot modules

Reply:
https://github.com/finos/symphony-bdk-java/pull/618/commits/3bfd6137089a2b1b5d09de71ce25734dc0aeaff4 done